src/devices/video/mc6845.cpp

```diff
@@ -769,7 +799,7 @@ TIMER_CALLBACK_MEMBER(mc6845_device::handle_line_timer)
 
 	/* Set VSYNC and DE signals */
 	set_vsync( new_vsync );
-	set_de( m_line_enable_ff ? true : false );
+	set_de( m_line_enable_ff && nonzero_horizontal_width ? true : false );
```

Comment:
Please use parentheses for arguments to the conditional operator, and since you’re editing the line anyway, change it to use “English” spacing around the partentheses, like:
```c++
	set_de((m_line_enable_ff && nonzero_horizontal_width) ? true : false);
```